build.gradle

```diff
@@ -959,22 +959,13 @@ if (usesShadowedDependencies.toBoolean()) {
     configurations.apiElements.outgoing.artifacts.clear()
     configurations.runtimeElements.outgoing.artifact(tasks.named("shadowJar", ShadowJar))
     configurations.apiElements.outgoing.artifact(tasks.named("shadowJar", ShadowJar))
-    tasks.named("jar", Jar) {
-        enabled = false
```

Comment:
This task has to stay disabled, or at least needs to be reconfigured to a different classifier like `-dev-preshadow` when shadow is enabled, otherwise you have 2 gradle tasks producing the same jar: `mod-dev.jar` when shadow is enabled. This can lead to issues with confused jars, or gradle erroring out if both tasks try to be run at build time.